src/main/functions/explode.scala

```diff
@@ -0,0 +1,8 @@
+package org.virtuslab.iskra.functions
+
+import org.apache.spark.sql
+import org.virtuslab.iskra.Column
+import org.virtuslab.iskra.types.{ ArrayOptType, DataType }
+
+def explode[T <: DataType](c: Column[ArrayOptType[T]]): Column[T] = Column(sql.functions.explode(c.untyped))
```

Comment:
We should have a way to prevent  users from using `explode` more than once in the same `select` clause as that would result in a runtime error. This constraint doesn't seem to be easy to express in the current model of iskra. However I'm in the middle of a major redesign of the library's model so I'll try to take this use case into account